testing
2015-06-06 10:55:16.531
ruby -v -W0 test/unit/test_run_test.rb
ruby 1.9.3p194 (2012-04-20 revision 35410) [x86_64-linux]
Run options: 

# Running tests:

..FEEEEEEE.EE-StandardError Error: #<NoMethodError: undefined method `log_path?' for #<TestRun:0x000000028e3b98>>
/home/greg/Desktop/src/Open-Table-Explorer/app/models/test_run.rb:258:in `run'
test/unit/test_run_test.rb:208:in `test_run'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
/usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
/usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
/usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
/usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
/usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
/usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
/usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
/usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
/usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
/usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
/usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
/usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
/usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
/usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
/usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'
E....

Finished tests in 0.633146s, 28.4294 tests/s, 42.6442 assertions/s.

  1) Failure:
test_error_score?(TestRunTest) [test/unit/test_run_test.rb:189]:
<nil> expected but was
<#<Unit:0x000000027c1d28
 @directories=["test/data_sources/"],
 @edit_files=[],
 @missing_files=
  ["app/models/.rb",
   "test/unit/_test.rb",
   "script/.rb",
   "test/integration/_test.rb",
   "test/long_test/_test.rb",
   "test/assertions/_assertions.rb",
   "test/unit/_assertions_test.rb",
   "log/library/.log",
   "log/assertions/.log",
   "log/integration/.log",
   "log/long/.log"],
 @model_basename=:"",
 @model_class_name=:"",
 @project_root_dir="">>.

  2) Error:
test_file_bug_reports(TestRunTest):
RuntimeError: path=nil must be a string
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/file_pattern.rb:52:in `path2model_name?'
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/unit.rb:12:in `new_from_path?'
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/test_executable.rb:41:in `log_path?'
    test/unit/test_run_test.rb:101:in `test_file_bug_reports'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

  3) Error:
test_log_file(TestRunTest):
NoMethodError: undefined method `log_file' for #<TestExecutable:0x0000000296a440>
    test/unit/test_run_test.rb:71:in `test_log_file'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

  4) Error:
test_log_passed?(TestRunTest):
ArgumentError: wrong number of arguments (0 for 1)
    test/unit/test_run_test.rb:147:in `test_log_passed?'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

  5) Error:
test_log_path?(TestRunTest):
NoMethodError: undefined method `log_path?' for #<Repository:0x00000002e3f010>
    test/unit/test_run_test.rb:53:in `test_log_path?'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

  6) Error:
test_new_from_pathname(TestRunTest):
NameError: undefined local variable or method `executable_file_file' for #<TestRunTest:0x00000002621bd0>
    test/unit/test_run_test.rb:48:in `test_new_from_pathname'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

  7) Error:
test_parse_header(TestRunTest):
RuntimeError: path=nil must be a string
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/file_pattern.rb:52:in `path2model_name?'
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/unit.rb:12:in `new_from_path?'
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/test_executable.rb:41:in `log_path?'
    test/unit/test_run_test.rb:154:in `test_parse_header'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

  8) Error:
test_parse_log_file(TestRunTest):
Errno::ENOENT: No such file or directory - /1.9/1.9.3p194/quiet/test_run.log
    test/unit/test_run_test.rb:121:in `read'
    test/unit/test_run_test.rb:121:in `test_parse_log_file'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

  9) Error:
test_ruby_run_and_log(TestRunTest):
NoMethodError: undefined method `error_score?' for #<Repository:0x00000002e3f010>
    test/unit/test_run_test.rb:83:in `test_ruby_run_and_log'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

 10) Error:
test_ruby_test_string(TestRunTest):
NoMethodError: undefined method `ruby_test_string' for #<Repository:0x00000002e3f010>
    test/unit/test_run_test.rb:58:in `test_ruby_test_string'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

 11) Error:
test_run(TestRunTest):
NoMethodError: undefined method `output' for nil:NilClass
    test/unit/test_run_test.rb:208:in `test_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:1058:in `run'
    /usr/lib/ruby/1.9.1/test/unit/testcase.rb:17:in `run'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:36:in `block in run'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:403:in `_run__1809415215368435636__setup__3729975415393423492__callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:405:in `__run_callback'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:385:in `_run_setup_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/callbacks.rb:81:in `run_callbacks'
    /usr/lib/ruby/vendor_ruby/active_support/testing/setup_and_teardown.rb:35:in `run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:823:in `block in _run_suite'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `map'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:815:in `_run_suite'
    /usr/lib/ruby/1.9.1/test/unit.rb:565:in `block in _run_suites'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `each'
    /usr/lib/ruby/1.9.1/test/unit.rb:563:in `_run_suites'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:775:in `_run_anything'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:964:in `run_tests'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:951:in `block in _run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `each'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:950:in `_run'
    /usr/lib/ruby/vendor_ruby/minitest/unit.rb:939:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:21:in `run'
    /usr/lib/ruby/1.9.1/test/unit.rb:326:in `block (2 levels) in autorun'
    /usr/lib/ruby/1.9.1/test/unit.rb:27:in `run_once'
    /usr/lib/ruby/1.9.1/test/unit.rb:325:in `block in autorun'

18 tests, 27 assertions, 1 failures, 10 errors, 0 skips

